src/main/kotlin/net/atos/zac/app/admin/model/RESTZaaktypeInrichtingscheck.kt

```diff
@@ -0,0 +1,36 @@
+/*
+ * SPDX-FileCopyrightText: 2022 Atos, 2025 Lifely
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+package net.atos.zac.app.admin.model
+
+/**
+ * 5 statustype; Intake, In behandeling, Heropend, Wacht op aanvullende informatie,
```

Comment:
I have no idea what this description is about.. I think we should just remove it?